bedrock/firefox/redirects.py

```diff
@@ -515,4 +517,7 @@ def firefox_mobile_faq(request, *args, **kwargs):
     redirect(r'^hello/?$', 'https://support.mozilla.org/kb/hello-status'),
     redirect(r'^firefox/hello/?$', 'https://support.mozilla.org/kb/hello-status'),
     redirect(r'^firefox(?:\/\d+\.\d+(?:\.\d+)?(?:a\d+)?)?/hello/start/?$', 'https://support.mozilla.org/kb/hello-status'),
+
+    # bug 1299947
+    redirect('^firefox/channel/$', 'firefox.channel.desktop'),
```

Comment:
Should probably make this trailing slash optional? (Sorry I missed during my first review.)